lib/active_zuora/generator.rb

```diff
@@ -29,6 +29,7 @@ def generate_classes
           class_name = complex_type.attribute("name").value
           # Skip the zObject base class, we define our own.
           next if class_name == "zObject"
+          class_name[0] = class_name[0].upcase
```

Comment:
@jconley88 you could use `String#capitalize` here on the whole string
http://ruby-doc.org/core-2.2.3/String.html#method-i-capitalize